src/ms_io.py

```diff
@@ -331,18 +359,102 @@ def read_psms(filename: str) -> pd.DataFrame:
 
 
 def _read_psms_mztab(filename: str) -> pd.DataFrame:
```

Comment:
By the way there is an mzTab reader in Pyteomics that produces DataFrames. You may find it useful here.

Reply:
Cool, that's probably a (somewhat) recent addition then? Otherwise I missed it the first time a wrote my own mzTab code a few years ago. I'll use the Pyteomics functionality then instead.

Reply:
Yep, added in the beginning of 2019 (also it was missing from the docs for a year due to a bug that I very recently found :man_facepalming: ).

Reply:
Good to know. I have a few more projects where I can start using it then. 🙂